### PR TITLE
Feature/finalise button

### PR DESCRIPTION
**Pull request by MKaHead**
Not really final, because we need more core feature ready. But l like to merge the actual state in develop-v2, because it's more stable than before.

### CODE REVIEW

**Comment by MKaHead** [comment]
fixed svg stuff is only a mock for the moment

File: src/components/molecules/button/index.js [hunk 1]
```diff
@@ -2,24 +2,40 @@ import { LitElement, html, css, unsafeCSS } from 'lit-element';
 import { classMap } from 'lit-html/directives/class-map';
 import buttonCSS from './index.scss';
 
+const icon = () => html`
```
Comment (LucaMele) [inline src/components/molecules/button/index.js]: why not `import` our new icon? and is it ok for @jojackpot to have a different arrow?

File: src/components/molecules/button/index.js [hunk 2]
```diff
@@ -2,24 +2,40 @@ import { LitElement, html, css, unsafeCSS } from 'lit-element';
...
-      ghost: { type: Boolean },
+      secondary: { type: Boolean },
+      large: { type: Boolean },
+      inverted: { type: Boolean },
+      cta: { type: Boolean },
       motion: { type: Boolean },
       disabled: { type: Boolean },
 
-      onClick: { type: Function },
+      onClick: { type: String },
```
Comment (LucaMele) [inline src/components/molecules/button/index.js]: on line 48 we check for type function but here is string.. is this intended as not finished code or a n error?

**Comment by MKaHead** [comment]
- fix button property
- change molecule to atom
- place the svg mock somewhere else but couldn't fix the initial problem in this branch. The svg (build, example...) stuff should be fixed in a different branch.

**Comment by jackblackCH** [comment]
LGTM

**Pull request merged**